packages/build/src/plugins/spawn.ts

```diff
@@ -50,11 +48,7 @@ const startPlugin = async function ({ pluginDir, nodePath, buildDir, childEnv, f
     execPath: nodePath,
     env: childEnv,
     extendEnv: false,
-    // Feature flag: https://app.launchdarkly.com/default/production/features/netlify_build_use_json_serialization_for_plugin_ipc/targeting
-    // TODO: remove feature flag once fully rolled out
-    ...(!featureFlags.netlify_build_use_json_serialization_for_plugin_ipc && {
-      serialization: 'advanced',
-    }),
+    serialization: 'advanced',
```

Comment:
Shouldn't this be removed? We want NON-advanced serialization, right?

Reply:
You are absolutely right! Thanks for looking that carefully!